src/query/api/v1/handler/prometheus/handleroptions/service_options.go

```diff
@@ -42,6 +42,11 @@ const (
 	// of propagating aggregator placement to clients, usually needed when there is
 	// a large amount of clients sending traffic to m3aggregator.
 	defaultM3AggWarmupDuration = 0
+
+	// DefaultServiceEnvironment is the default service ID environment.
+	DefaultServiceEnvironment = "default_env"
```

Comment:
Are these already declared somewhere else or were they moved?

Reply:
Yeah, it was moved from `headers.go`; moved it back :+1: